### ISSUE TITLE
Manually fix columns width, define columns' width as (non) flexible

### ISSUE THREAD

**Issue by ilippert** [JabRef/jabref#6348]
JabRef 5.1--2020-04-23--bbea7df
Linux 5.5.17-200.fc31.x86_64 amd64 
Java 14.0.1

I'm always frustrated when I open jabref and the columns have columns widths that I have to fiddle with. 

**Describe the solution you'd like**
I want to have a slightly more (advanced/extended) entry table columns specification in the Preferences. At the moment that preference panel only configures the Name of the column. I want to see three configurable columns

- Name (as is)
- width (in percentage or something like pixels
- fix width (checkboxes that when activated fix that column width)

I believe that this improvement would also reinstate a feature that was enabled in an earlier incarnation of jabref: define default widths for columns and retrieve values from the current layout of the columns.

**Additional context**

**Comment by ilippert** [comment]
This is clearly related to the bug https://github.com/JabRef/jabref/issues/6259

**Comment by ilippert** [comment]
This enhancement is also related to https://github.com/JabRef/jabref/issues/6626 https://github.com/JabRef/jabref/issues/6818 https://github.com/JabRef/jabref/issues/6690

**Comment by ilippert** [comment]
I would still like this feature

**Comment by protist** [comment]
To add a bit more detail, any manual changes to column widths is not saved between sessions. To replicate:

1. Change column width
2. Save and quit
3. Launch JabRef again

Expected behaviour: columns should have the same width as previously defined.
Observed behaviour: columns reset to the default width.

This is particular annoying for fields such as "Year", which barely require any width, but take space away from more verbose columns like "Title" or "Comment".

**Comment by samuelVJrobinson** [comment]
I agree with the previous comments. This would be extremely useful to have in future versions of JabRef.

**Comment by ilippert** [comment]
JabRef 5.4--2021-09-22--e9d845b
Linux 5.13.16-200.fc34.x86_64 amd64 
Java 16.0.2 
JavaFX 17+18

just to reiterate another part of the problem: when I change the width of the column, JabRef unpredictably shifts other columns' widths.

**Comment by Siedlerchr** [comment]
Did you disable "Fit columns on screen"?

**Comment by ilippert** [comment]
> Did you disable "Fit columns on screen"?

I have activated "Fit table horizontally on screen"

**Comment by ThiloteE** [comment]
Yes, disabling that one could help you. I don't use it either. Widths of Columns are saved after closing and opening of JabRef and it doesn't lead to weird problems, when resizing the group column or trying to manually adjust the widths of columns. I was soooo close to posting a video about this, but then i found this option and i was like hmmmm... maybe other people like it xD

Out of curiosity, Are there any statistics how many users of JabRef use or disable this function?

**Comment by ilippert** [comment]
Thanks for the suggestion. Deactivating the `Fit` immediately helps.

**Comment by shmilee** [comment]
"Fit table horizontally on screen" helps.
But the "groups interface" width  is still not saved.
